src/finch/compiled.py

```diff
@@ -2,10 +2,66 @@
 from functools import wraps
 
 from .julia import jl
-from .tensor import Tensor
```

Comment:
We can't import it top level anymore?

Reply:
It led to a recursive import; anyway it's undone in https://github.com/finch-tensor/finch-tensor-python/pull/83/commits/87f384c06d507d9c9024d880b347977fa7a14a5f.